### ISSUE TITLE
'Agent connection server' problem when starting javaws jnlp application

### ISSUE THREAD

**Issue by rafaelgui** [nokia/RED#330]
We are using robotframework-FormsLibrary since RED 0.7.2  https://github.com/dvanherbergen/robotframework-formslibrary  ( based on remoteswinglibrary )

Since RED 0.7.7 we receive a 'Problem Occured' popup when we run tests with jnlp.
Detail : An internal error occurred during: "Agent connection server". Cannot cast java.lang.Boolean to java.lang.String
![image](https://user-images.githubusercontent.com/22100148/64103063-910c1f00-cd71-11e9-9f4a-834dec3065fb.png)

With RED 0.7.6, we don't have the problem.

Currently we are using Eclipse IDE for Java Developers
Version: 2019-03 (4.11.0)
Build id: 20190314-1200
Python 3.7

Sample code / jnlp
[demo.zip](https://github.com/nokia/RED/files/3565344/demo.zip)

**Comment by michalanglart** [comment]
Hi,

1) Do you experience this problem in newer RED? Version 0.7.7 was released around 2.5 year ago.
2) Can you attach exceptions logged into your `metadata/.log` file?

**Comment by rafaelgui** [comment]
Hello,

1. Yes, we alse experience this problem in the latest version 0.8.13.
2.
[log.txt](https://github.com/nokia/RED/files/3569160/log.txt)

**Comment by michalanglart** [comment]
Ok, thanks for the .log file - I was able to reproduce the issue with simple setup. We'll provide a fix in next version which is about to be released.

**Comment by michalanglart** [comment]
[GH-330.zip](https://github.com/nokia/RED/files/3569289/GH-330.zip)

Attached zip contains minimal project which reproduces the issue. The root cause is that there is a `BuiltIn.Import Library` keyword used with arguments other than strings inside other library, like here:

```python
    def import_lib_one(self):
        from robot.libraries.BuiltIn import BuiltIn
        BuiltIn().import_library('lib1', 42)
```

with the setup as above RED gets `library_event` from [listener](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#listener-version-2) where `args` list contains Integer (Boolean/whatever) instead of String but tries to cast it.

It seems that it is not possible to reproduce it by using normal library imports (in `Settings` table) or `BuiltIn.Import Library` keyword from inside of `.robot` file.

**Comment by michalanglart** [comment]
Fixed in 0.9.0
